botocore/data/lambda/2015-03-31/examples-1.json

```diff
@@ -1,1513 +1,5 @@
 {
   "version": "1.0",
   "examples": {
-    "AddLayerVersionPermission": [
```

Comment:
This is deleting all of our Lambda examples. We may want to double check on this before merging.